app/src/main/java/com/github/orkest/bluetooth/data/BluetoothServiceManager.kt

```diff
@@ -72,14 +81,130 @@ class BluetoothServiceManager : BluetoothInterface {
 
     }
 
-    override fun connectDevice() {
-        TODO("Not yet implemented")
+    @SuppressLint("MissingPermission")
```

Comment:
All good to me. Careful to not move the permission mechanism from the activity otherwise it would break this part.

Reply:
Yes you are right thank you!